inc/classes/class_db_mysql.php

```diff
@@ -5,17 +5,17 @@ class db
     /**
      * @var int
      */
-    public $link_id = 0;
+    private $link_id = 0;
```

Comment:
used in class_import.php line 808

Reply:
This is also changed in this PR. See https://github.com/lansuite/lansuite/pull/190/files#diff-07003def8a09985dfaa2f74d4cbcb1a2